frontend/tailwind.config.js

```diff
@@ -11,19 +11,17 @@ export default {
         xl: "1200px",
       },
       colors: {
-        // Dark base background color
-        ground: "#0C0F0B",
-
         // Theme colors
         primary: "#7FEE64",
-        "accent-pink": "#FF0ECA",
-        "accent-blue": "#B8E4FF",
```

Comment:
One benefit of putting these examples in the monorepo is that we don't have to duplicate this every time.

Reply:
I wonder if we should consider moving turbo.art over? (doesn't have to be right now, but a linear ticket can be made if we think so)

Reply:
Yeah I was thinking about this too. I think Sona wants to create more branded examples and it would be helpful to have access to tokens + such.

Reply:
I don't think we should do this now just food for thought